Userland/Utilities/markdown-check.cpp

```diff
@@ -176,8 +176,19 @@ RecursionDecision MarkdownLinkage::visit(Markdown::Text::LinkNode const& link_no
             return RecursionDecision::Recurse;
         }
         if (url.scheme() == "help") {
-            // TODO: Check that the man page actually exists. (That check would also fail because we are currently referring to some nonexistent man pages.)
-            outln("Not checking man page link {}", href);
+            if (url.host() != "man") {
+                warnln("help:// URL without 'man': {}", href);
+                append_invalid_link();
+                return RecursionDecision::Recurse;
+            }
+            if (url.paths().size() < 2) {
+                warnln("help://man URL is missing section or page: {}", href);
+                append_invalid_link();
+                return RecursionDecision::Recurse;
+            }
+            auto file = String::formatted("../man{}/{}.md", url.paths()[0], url.paths()[1]);
+
+            append_valid_link({ file, String(), StringCollector::from(*link_node.text) });
```

Comment:
these functions do not exist

Reply:
apparently yes, I am too dumb to rebase.